examples/mapping_example.py

```diff
@@ -0,0 +1,113 @@
+#%%
+from math import isclose
+from pathlib import Path
+
+import numpy as np
+from mapping_functions import *
+
+
+def calculated_as_expected(expected, calculated):
+    ok = []
+    for i in range(len(expected)):
+        ok.append(isclose(expected[i], calculated[i], rel_tol=0.001, abs_tol=0.0))
+    return all(ok)
```

Comment:
is it necessary to store the array "ok"?  from the looks of it, you could do:
for i in range(len(expected)):
        if not isclose(expected[i], calculated[i], rel_tol=0.001, abs_tol=0.0):
                 return False
return True

Reply:
it is not indeed, changed it into your suggestion